### PR TITLE
Bugfix/fix lambda

### PR DESCRIPTION
**Pull request by vikicybr**
Converting PtaCloudTrailToSns lambda from javascript to python

### CODE REVIEW

File: src/PtaCloudTrailToSns/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# AWS SDK clients for Python
+s3 = boto3.client('s3')
+sns = boto3.client('sns', region_name=os.environ['SNSREGION'])
+
+def publish_record(record):
+    print('Publishing notification:', record)
+    sns.publish(Message=json.dumps(record), TopicArn=os.environ['SNSTOPICARN'])
+
+def lambda_handler(event, context):
+    print(json.dumps(event))
```
Comment (duekm) [inline src/PtaCloudTrailToSns/lambda_function.py]: print what (is there PII / sanitized)? where?

Comment (vikicybr) [reply src/PtaCloudTrailToSns/lambda_function.py]: This is what was there before

**Review by duekm — APPROVED** [review]
nice :-)

File: src/PtaCloudTrailToSns/lambda_function.py (hunk 1 above)
Comment (duekm) [reply src/PtaCloudTrailToSns/lambda_function.py]: changed to logger level

**Pull request closed without merging**